src/cf-forms/usage.md

```diff
@@ -331,7 +333,9 @@ Generally this is only useful for documentation purposes.
            type="text"
            id="full-textinput-example"
            value="Lorem ipsum">
-    <br><br>
+</div>
+<br>
```

Comment:
For the code example, should we omit the `<br>` and just have a blank line here?

Reply:
It's from way back before these were ever wrapped. I'd say yes

Reply:
Does not work:
![screen shot 2017-10-03 at 6 00 44 pm](https://user-images.githubusercontent.com/704760/31151299-d12bfcba-a864-11e7-9cc7-7f1e102004ff.png)